composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.2",
+        "ezsystems/ezpublish-kernel": "^6.2 || ^7.0",
```

Comment:
> Inconsistent space usage compared to the line below, but I don't know which is correct..

It seems none :) This probably is due to some changes in composer.json format. Right now [doc mentions only double pipe `||` surrounded by spaces](https://getcomposer.org/doc/articles/versions.md#version-range), so I aligned this with it.
Thanks for pointing this out @glye